src/WebJobs.Script/Description/Node/Compilation/TypeScript/TypeScriptCompilationService.cs

```diff
@@ -1,36 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.Azure.WebJobs.Script.Config;
-
-namespace Microsoft.Azure.WebJobs.Script.Description.Node.TypeScript
-{
-    public class TypeScriptCompilationService : ICompilationService<IJavaScriptCompilation>
```

Comment:
So we don't support type script anymore?

Reply:
cc: @mhoeger 
NodeJS worker needs to handle this.